packages/locales/lib/human/de.json

```diff
@@ -763,5 +763,23 @@
   "enter_translation": "Übersetzung eingeben",
   "individual_filters": "individuell gefiltert",
   "best_buddy": "Bester Kumpel",
-  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}"
+  "data_limit_reached": "Du hast kürzlich zu viele Daten angefordert und befindest dich im Cooldown bis {{until}}",
+  "unknown_station": "unbekannte Kraftquelle",
+  "exclude_battle": "Dyna-Raids ausschließen",
+  "station": "Kraftquelle",
+  "stations": "Dynamax",
+  "stations_options": "Kraftquellen Optionen",
+  "all_stations": "alle Kraftquellen",
+  "search_battles": "Suche Dyna-Raids",
```

Comment:
Why not being consistent with search phrases? :)

Reply:
It's the same in English. Stations/Battles used in backend, translated to Power Spots/Dynamax or Max Battles in clients.

Reply:
@TurtIeSocks was more about the translation itself, he changed the other search_key values, but this matches the old phrasing :) 
it's not about dyna/spots or something else :)

Reply:
I left it generic in this case. In English it's just `Override` without any category attached to it. Dynamax or Max Battles as a prefix was just too long and feels unnecessary with the way the UI is setup.